packages/prompts/src/index.ts

```diff
@@ -483,7 +610,8 @@ export const group = async <T>(
 	const promptNames = Object.keys(prompts);
 
 	for (const name of promptNames) {
-		const result = await prompts[name as keyof T]({ results }).catch((e) => {
+		const prompt = prompts[name as keyof T];
+		const result = await prompt({ results })?.catch((e) => {
```

Comment:
Allows `p.group({ something: () => void })` to be skipped

Reply:
Yep, this allows you to conditionally skip by returning `undefined`.

Agreed on the weird `catch => throw`, will have to think about that some more.